packages/scxa-tsne-plot/src/ClusterTSnePlot.js

```diff
@@ -205,6 +207,7 @@ ClusterTSnePlot.propTypes = {
   plotData: PropTypes.shape({
     series: PropTypes.array.isRequired
   }),
+  initialCellTypeValues: PropTypes.array.isRequired,
```

Comment:
This was the design decision I made. You are right that we can make them `default props` in `ClusterTSnePlot` but in the future, if we need to pass them as `props` then they would need to come from `TSnePlotView` as that renders `ClusterTSnePlot` so would need to have this as prop in`TSnePlotView` as well. So I decided that it would be better that we keep the `default props` in `TSnePlotView` so that in future if we need to change them then we don't have to change any code in the front-end component.